python/datashaper/datashaper/table_store.py

```diff
@@ -23,13 +29,13 @@ class ColumnStats:
     distinct: int
     invalid: int
     mode: Any
-    min: Optional[float] = None
-    max: Optional[float] = None
-    mean: Optional[float] = None
-    median: Optional[float] = None
-    stdev: Optional[float] = None
-    bins: Optional[List[Bin]] = None
-    categories: Optional[List[Category]] = None
+    min: float | None = None

```

Comment:
I feel like we're going back and forth on these- I keep switching them to optional